core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -383,6 +383,28 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
     {% endif %}
   </div>
 
+  <script type="text/ng-template" id="modals/setDashboard">
+    <div class="modal-body oppia-set-dashboard-modal">
+      <div ng-if="!defaultDashboardSelected">
+        <p style="padding-top: 30px; padding-bottom: 20px; text-align: center;">
+          Hi! We recently launched the learner dashboard. Keeping track of what you've learned will be easier than ever!
```

Comment:
I know we had already agreed on this language in a previous meeting, but I'm wondering now if we should change it a bit. There will be users seeing this who are using Oppia for the first time, and I'm not sure we currently have plans to change the text in the future, so mentioning that we recently launched it might not make the most sense in every context. Let's discuss a bit more during our Gitter meeting tomorrow? 

cc @rachelwchen